### DISCUSSION TITLE
Revit API Extensions

### DISCUSSION THREAD

**Discussion by Nice3point** [jeremytammik/RevitLookup#125] [Show and tell]
Hi guys, it's time to pump Revit Api, I started developing a library that will make it easier to write code using extensions.
In general, instead of `Method(Method(Method(Method(Method()))))`, you can write `Method.Method.Method.Method.Method`. And of course, I added a couple of new methods and overloads that are not in the API. Working with the Ribbon and *Utils classes has been greatly simplified. If you have any suggestions for improving the API, write to me about it

https://github.com/Nice3point/RevitExtensions

**Comment by jeremytammik** [comment]
Wow, looks great, great potential boost in legibility and productiveness. I'll share it on the blog asap.
